### ISSUE TITLE
Possible bug in ESST4B

### ISSUE THREAD

**Issue by janlav** [OpenIPSL/OpenIPSL#150]
I noticed on [OMs OpenIPSL test](https://libraries.openmodelica.org/branches/master/OpenIPSL/OpenIPSL.html) that [Examples/.../ESST4B](https://github.com/OpenIPSL/OpenIPSL/blob/master/OpenIPSL/Examples/Controls/PSSE/ES/ESST4B.mo) didn't compile and saw some strange things:

1.  The test compiles and runs under Dymola but both limiters are at-limit all of the time so that the only part that does something to the excitation voltage is the "Rectifier Commutation Voltage Drop" , it seems to be bad test of the system. Anyone agree?
2. The voltage is 1.56 pu at the generator, but the PF-solution say that it should be unity! This is likely what OM fails to compute, but it is probably not the solution to the initial problem that we should want.
3. The sensed complex current seems to be wrong:
 ` V_T = Gen_terminal.vr + j*Gen_terminal.vi;`
  `I_T = Gen_terminal.ir - Bus.ir + j*(Gen_terminal.ii - Bus.ii);`
This seems to be **doubling** the real and imaginary components of the current as it is used in the ESST4B-example.

Both the example and model should be investigated.

**Comment by marcelofcastro** [comment]
I'm taking over this issue

**Comment by dietmarw** [comment]
I just ran the example and can confirm 3. . The OpenModelica issue (2.) seems no longer valid as I get the the expected 1 pu voltage value.

**Comment by dietmarw** [comment]
Thinking about 3. a bit  ... the equation for I_T is simply wrong. It should be:
```diff
- I_T = Gen_terminal.ir - Bus.ir + j*(Gen_terminal.ii - Bus.ii);
+ I_T = Gen_terminal.ir + j*Gen_terminal.ii;
```

**Comment by marcelofcastro** [comment]
So, (1.) and (2.) were basically related to the initialization of the exciter and due to some connections that were wrong. This is solved already. About (3.), @dietmarw is right and we just need to replace the equation.

**Comment by dietmarw** [comment]
OK I'll provide a fix as soon as #228  is merged.
